src/doc/src/reference/specifying-dependencies.md

```diff
@@ -56,7 +56,8 @@ using special operators, though it shouldn't be necessary most of the time.
 ### Caret requirements
 
 **Caret requirements** are an alternative syntax for the default strategy,
```

Comment:
You're starting a new sentence in the next line, so the comma at the end of this one should be turned into a period.